Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatUnreadMessagesCountDecorationView.swift

```diff
@@ -15,7 +15,7 @@ open class ChatThreadRepliesCountDecorationView: ChatMessageDecorationView, Appe
     /// The container that the contentTextLabel will be placed aligned to its centre.
     open private(set) lazy var container: UIView = UIView()
         .withoutAutoresizingMaskConstraints
-        .withAccessibilityIdentifier(identifier: "repliesCountDecorationContainer")
+        .withAccessibilityIdentifier(identifier: "unreadMessagesDecorationContainer")
```

Comment:
This one will also need to be updated, as inside thread it won't represent unread messages but instead number of replies